modules/dxp/apps/portal-search-similar-results/portal-search-similar-results-web/src/main/java/com/liferay/portal/search/similar/results/web/internal/portlet/SimilarResultsPortletPreferencesImpl.java

```diff
@@ -120,20 +119,18 @@ public String getStopWords() {
 
 	@Override
 	public Float getTermBoost() {
-		Optional<String> optional = _portletPreferencesHelper.getString(
+		String stringValue = _portletPreferencesHelper.getString(
```

Comment:
@slnn Either `string` or `value` is enough for this variable.

Reply:
@slnn `string` seems better because it matches the method that returns the variable value.